ragna/source_storages/_vector_database.py

```diff
@@ -46,7 +46,7 @@ def requirements(cls) -> list[Requirement]:
             # to manage and mostly not even used by the vector DB. Chroma provides a
             # wrapper around a compiled embedding function that has only minimal
             # requirements. We use this as base for all of our Vector DBs.
-            PackageRequirement("chromadb<=0.5.11,>=0.4.13"),
+            PackageRequirement("chromadb<=0.6.1,>=0.4.13"),
```

Comment:
```suggestion
            PackageRequirement("chromadb>=0.6.0"),
```

Run `python scripts/update_optional_dependencies.py` afterwards to automatically update the `pyproject.toml`.

Reply:
[Done](https://github.com/Quansight/ragna/pull/530/commits/0c95e974c06e9b792f6abbcd779cb7a4972afb39)